index.js

```diff
@@ -53,6 +53,7 @@ var config = {
 	animateInParallel: false,
 	appIcon: null,
 	pathToModule: '',
+	autoCleanup: true, // Auto cleanup
```

Comment:
Please don't set it to autoCleanup. It would confuse existing users of this module. But i like the feature :+1: